.github/workflows/build-nightly.yml

```diff
@@ -81,10 +81,9 @@ jobs:
 
     - name: Build
       run: |
-        mkdir build
-        cd build
-        cmake .. -DCMAKE_BUILD_TYPE=Release
-        make -j $(nproc)
+        cmake -B build -DCMAKE_BUILD_TYPE=Release -D WICKED_LINUX_TEMPLATE=ON 
```

Comment:
Let's not change these commands, just specify that the template build is ON by default, like the other projects.

Reply:
Yes, I mean that it's set to ON in the Cmake, and don't modify the github build scripts

Reply:
Why?

Reply:
Because all the other projects work like that, and I prefer to not modify github builder if not necessary

Reply:
Don't worry, CMake's `-B` flag is just `mkdir` and `cmake --build` automatically finds what build system you're using to build. So if you build with **Unix Makefiles** it'll run `make`. If you use something like **Ninja** it will use that instead.

So this makes the CI more future proof, and can prevent errors.

If I change `CMakeSettings.txt` to build the template, that will definitely upset people since it's an unused build that increases global build time. For me it's not an issue, but someone with lower-end hardware would be affected and might not even know why.

What I did wasn't an actual change, just an abstraction for `mkdir` and `make`, which allows me to speed up build times for everyone and toggle settings, if we ever have to do that in the future. 

This won't affect the binaries or anything, I try my best to avoid intrusive changes to the CI.

Reply:
You can use cmake however you want in your projects, but the minimal cmake steps that I used and trust are highlighted in the readme and have been used in the github builder with success. All the projects so far work the same way, so I would like this new project follow this too. 

"that will definitely upset people since it's an unused build that increases global build time" - but it can be disabled.